automated install/basic-install.sh

```diff
@@ -1772,9 +1772,14 @@ finalExports() {
     echo "INSTALL_WEB_INTERFACE=${INSTALL_WEB_INTERFACE}"
     echo "LIGHTTPD_ENABLED=${LIGHTTPD_ENABLED}"
     echo "CACHE_SIZE=${CACHE_SIZE}"
-    echo "DNS_FQDN_REQUIRED=true"
-    echo "DNS_BOGUS_PRIV=true"
     }>> "${setupVars}"
```

Comment:
```suggestion
    echo "DNS_FQDN_REQUIRED=${DNS_FQDN_REQUIRED:-true}"
    echo "DNS_BOGUS_PRIV=${DNS_BOGUS_PRIV:-true}"
    }>> "${setupVars}"
```

These can just go back into here now, right?